src/routes/quote.js

```diff
@@ -7,6 +7,101 @@ const { TARGET_PRICE_DENOMINATOR } = require('../lib/constants');
 const router = express.Router();
 const { Zero } = ethers.constants;
 
+/**
+ * @openapi
+ * /v2/quote:
+ *   get:
+ *     tags:
+ *       - Quote
+ *     description: Get a quote for a cover
+ *     parameters:
+ *     - in: query
+ *       name: productId
+ *       required: true
+ *       schema:
+ *         type: integer
+ *         description: The product id
+ *     - in: query
+ *       name: amount
+ *       required: true
+ *       schema:
+ *         type: string
+ *         description: The cover amount
+ *     - in: query
+ *       name: period
+ *       required: true
+ *       schema:
+ *         type: integer
+ *         description: The cover period (in days)
+ *     - in: query
+ *       name: coverAsset
+ *       required: true
+ *       schema:
+ *         type: integer
+ *         description: The cover asset (e.g. 0 for ETH, 1 for DAI)
+ *     responses:
+ *       200:
+ *         description: Returns a quote object
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 quote:
+ *                   type: object
+ *                   properties:
+ *                     totalCoverAmountInAsset:
+ *                       type: string
+ *                       description: The total cover amount value in cover asset (smallest unit e.g. wei for ETH)
+ *                     annualPrice:
+ *                       type: string
+ *                       description: The annual price as a percentage value (2 decimals). Should be divided by 100.
```

Comment:
I believe the `annualPrice` is an integer value between 0-10000. i.e. 750 is 7.5%